src/Features/Core/Portable/Completion/Providers/AbstractSymbolCompletionFormat.cs

```diff
@@ -0,0 +1,153 @@
+        {
+            var format = SimplificationHelpers.PreferPredefinedTypeKeywordInDeclarations(options, context.SemanticModel.Language)
+                ? _displayFormatWithPredefinedTypes
+                : _displayFormatWithFrameworkTypes;
+
+            return GetMinimalText(symbol, context, options, format);
+        }
+
+        private string GetMinimalInsertionText(ISymbol symbol, SyntaxContext context, OptionSet options)
+        {
+            var format = SimplificationHelpers.PreferPredefinedTypeKeywordInDeclarations(options, context.SemanticModel.Language)
+                ? _insertionFormatWithPredefinedTypes
+                : _insertionFormatWithFrameworkTypes;
+
+            return GetMinimalText(symbol, context, options, format);
+        }
+
+        private string GetMinimalText(ISymbol symbol, SyntaxContext context, OptionSet options, SymbolDisplayFormat format)
+        {
+            string aliasText;
+            if (TryGetAliasText(symbol, format, context, out aliasText))
+            {
+                return aliasText;
+            }
+
+            var service = context.GetLanguageService<ISymbolDisplayService>();
+            var parts = service.ToMinimalDisplayParts(context.SemanticModel, context.Position, symbol, format);
+
+            if ((format.MiscellaneousOptions & SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers) != SymbolDisplayMiscellaneousOptions.None)
+            {
+                parts = Escape(context, parts);
+            }
+
+            return parts.ToDisplayString();
+        }
+
+        private Boolean TryGetAliasText(ISymbol symbol, SymbolDisplayFormat format, SyntaxContext context, out string text)
+        {
+            if (symbol is IAliasSymbol)
+            {
+                // Using SymbolDisplayFormat would result in somthing like "alias = type",
+                // so this needs to be special-cased here
+                text = (format.MiscellaneousOptions & SymbolDisplayMiscellaneousOptions.EscapeKeywordIdentifiers) != SymbolDisplayMiscellaneousOptions.None
+                    ? Escape(context, symbol.Name)
+                    : symbol.Name;
+                return true;
+            }
+
+            text = null;
+            return false;
+        }
+
+        public string GetInsertionTextAtInsertionTime(CompletionItem item, char? ch)
+        {
+            string insertionText = SymbolCompletionItem.GetInsertionText(item);
+
+            if (ch == _genericCommitChar)
+            {
+                var index = insertionText.IndexOf(_genericCommitChar);
+
+                if (index >= 0)
+                {
+                    return insertionText.Substring(0, index);
+                }
+            }
+
+            return insertionText;
+        }
+
+        private ImmutableArray<SymbolDisplayPart> Escape(SyntaxContext context, ImmutableArray<SymbolDisplayPart> parts)
```

Comment:
After further thought, I think it might be best to open a separate issue for this, so it can be considered independently. I'll back our the product changes, but leave the tests and mark them as skipped.